### PR TITLE
Branch quiz result

### PR DESCRIPTION
**Pull request by Auxinnn**
Update feature of quiz on storage and selection of questions. Can be merged after reviewed.

### CODE REVIEW

File: src/main/java/seedu/address/model/quiz/QuizResult.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof QuizResult)) {
+            return false;
+        }
+
+        QuizResult otherQuizResult = (QuizResult) other;
+        return otherQuizResult.getQuestionBody().equals(getQuestionBody());
```
Comment (Parcly-Taxel) [inline src/main/java/seedu/address/model/quiz/QuizResult.java]: You _can_ have duplicate question bodies in the quiz results – I was getting the unnecessary error `Quiz result contains duplicate results.`

This line should be `return otherQuizResult.getQuizTime().equals(getQuizTime());`.

**Review by Parcly-Taxel — CHANGES_REQUESTED** [review]
Only 3 quiz results are stored in the JSON file as of this moment. This arbitrary limit should be removed.

The quiz questions, when they are displayed, are also in the _fourth_ column of the application right now – bring those to the second column!

File: src/main/java/seedu/address/model/quiz/QuizResult.java (hunk 1 above)
Comment (Auxinnn) [reply src/main/java/seedu/address/model/quiz/QuizResult.java]: Sure. I will modify that. Thank you very much for the help. For the display question I will create a new box for it, so I will just temporarily leave it at fourth column for v1.2 and I will solve it in v1.3.

Comment (Auxinnn) [reply src/main/java/seedu/address/model/quiz/QuizResult.java]: Sure, I will modify that. Thank you very much for the help. For the display of quiz questions, I will create a new Ui window to display all quiz questions. So I will leave it at fourth column temporarily for v1.2 and implement further on v1.3.

**Review by ShirleyWangxt — COMMENTED** [review]
Seems fine in general

**Comment by shuiyao-sg** [comment]
@Auxinnn Have you made the changes requested by @Parcly-Taxel?

**Comment by Auxinnn** [comment]
@shuiyao-sg Yes I have made the changes.

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 2]
```diff
@@ -64,6 +64,12 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
             Command command = quizParser.parseCommand(commandText);
             commandResult = command.execute(model);
 
+            try {
```
Comment (irene-lly) [inline src/main/java/seedu/address/logic/LogicManager.java]: I think there is a problem with saving the addressbook this way. Try doing the quiz once and closing the application then run the application again and type a command e.g. "listnote" exit the application and all quizResults will be erased.

Comment (Auxinnn) [reply src/main/java/seedu/address/logic/LogicManager.java]: Oh yea right. I will try to fix that.

**Review by Parcly-Taxel — APPROVED** [review]

**Pull request merged**